hw03.ipynb

```diff
@@ -0,0 +1,120 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Overwriting hw03-1.py\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%file hw03-1.py\n",
+    "from mrjob.job import MRJob\n",
+    "import csv\n",
+    "class avg_word_count (MRJob):\n",
+    "    def Mapper (self,_,line):\n",
```

Comment:
```Traceback (most recent call last):
  File "hw3.py", line 14, in <module>
    avg_word_count.run()
  File "/Users/enkeboll/.pyenv/versions/3.8.6/lib/python3.8/site-packages/mrjob/job.py", line 616, in run
    cls().execute()
  File "/Users/enkeboll/.pyenv/versions/3.8.6/lib/python3.8/site-packages/mrjob/job.py", line 687, in execute
    self.run_job()
  File "/Users/enkeboll/.pyenv/versions/3.8.6/lib/python3.8/site-packages/mrjob/job.py", line 636, in run_job
    runner.run()
  File "/Users/enkeboll/.pyenv/versions/3.8.6/lib/python3.8/site-packages/mrjob/runner.py", line 495, in run
    raise ValueError('Job has no steps!')
ValueError: Job has no steps!```

mrjob needs the class functions `mapper()` and `reducer()`, not `Mapper()` and `Reducer()`

-3